build/Dockerfile-go1.23.7-rpmbuild-gcc7

```diff
@@ -1,7 +1,7 @@
-FROM quay.io/centos/centos:stream9
+FROM kaiachain/circleci-rpmbuild:1.23.0-gcc11
```

Comment:
Where is the Dockerfile for `kaiachain/circleci-rpmbuild:1.23.0-gcc11`? The new Dockerfile has an unknown recipe now. It has to be open sourced so you can theoretically build images without kaiachain's pre-built image.

Is 1.23.0 image also based on stream9? or is it based on centos7? Why missing bashrc issue appeared?